internal/listtoken/list_token.go

```diff
@@ -387,8 +387,6 @@ func (tk *Token) Validate(
 			return errors.New(ctx, errors.InvalidParameter, op, "list token's refresh component missing last item ID")
 		case st.LastItemUpdateTime.After(time.Now()):
 			return errors.New(ctx, errors.InvalidParameter, op, "list token's refresh component's last item was updated in the future")
-		case st.LastItemUpdateTime.Before(tk.CreateTime):
```

Comment:
Validating that the update time of the last item is after the create time of the token cannot account for items whose update time are earlier than the original query because of the added database transaction timeout buffer.

Reply:
Eh this check was a bit overzealous anyway, I don't think it adds that much confidence. We do have a variable for the offset but I'd still prefer we don't do this check.